File: Hibernatfe Java HW/Middleware/src/main/java/domain/ChatMessage.java
package domain;

import java.io.Serializable;

public class ChatMessage implements Serializable {
    public String message;
    public String sender;
    public ChatMessage(String message, String sender){
        this.message = message;
        this.sender = sender;
    }
    public ChatMessage(){
        this.message = null;
        this.sender = null;
    }
}

File: Crossplatform/c_sharp_version/Middleware/domain/ChatMessage.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Middleware.domain
{
    [System.Serializable]
    public class ChatMessage
    {
        public string message;
        public string sender;
       
        public ChatMessage(string message,string sender)
        {
            this.message = message;
            this.sender = sender;
        }
        public ChatMessage()
        {
            this.message = null;
            this.sender = null;
        }
    }
}
